keras/src/backend/common/variables.py

```diff
@@ -30,18 +30,17 @@ class KerasVariable:
             dtype type (`"float32"` if never configured).
         trainable: Optional. Boolean indicating if variable is trainable.
             Defaults to `True`.
+        autocast: Optional. Boolean indicating whether the variable supports
+            autocasting. If `True`, the layer may first convert the variable
+            to the compute data type when accessed. Defaults to `True`.
+        aggregation: Optional. String specifying how a distributed variable will
+            be aggregated. This argument is specific to TensorFlow. For more
```

Comment:
Is that accurate -- is it specific to TensorFlow? It's more of a semantic annotation, to be taken into account by downstream users. For instance if you write your own low-level distribution code in any backend you will need to look at it.

Reply:
Got it. Updated!